packages/react-html/src/manager.ts

```diff
@@ -0,0 +1,23 @@
+import {getSerializationsFromDocument} from './utilities';
+
+export default class Manager {
+  private isServer = typeof document === 'undefined';
+  private serializations = getSerializationsFromDocument();
+
+  set(id: string, data: unknown) {
+    if (this.isServer) {
+      this.serializations.set(id, data);
+    }
+  }
+
+  get<T>(id: string): T | undefined {
+    return this.serializations.get(id) as T | undefined;
+  }
+
+  extract() {
+    return [...this.serializations.entries()].map(([id, data]) => ({
```

Comment:
Why do we need to clone the entries array? I think you can just drop the clone and do `.entries().map`

Reply:
`Map.prototype.entries` returns an iterator, not an array.

<img width="872" alt="screen shot 2018-11-20 at 10 05 13 am" src="https://user-images.githubusercontent.com/3012583/48782263-d3b1ae80-ecab-11e8-9f22-ee3a9ad732f7.png">